Description: Set automake option subdir-object to remain forward compatible.
Author: Bas Couwenberg <sebastic@xs4all.nl>
Last-Update: 2013-09-30
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,4 +1,6 @@
 
+AUTOMAKE_OPTIONS = subdir-objects
+
 SUBDIRS = headers \
 	gaiaaux \
 	gaiaexif \
